### ISSUE TITLE
Hot reload

### ISSUE THREAD

**Issue by Fazerty** [jhipster/generator-jhipster#1423]
I'm using netbeans and I recently updated my project with jhipster 2.7.
I'm not able to make hot reload working (in debug mode with dev profile or with mvn spring-boot:run).
Have I to modify the pom.xml and how?
Is there a wiki that explain how to make it work?
Can I try this http://tux2323.blogspot.be/2014/06/hot-reload-support-with-spring-loaded.html
What are the differences between spring-loaded and jhipster-loaded?

Many Thanks

**Comment by jdubois** [comment]
[JHipster loaded](https://github.com/jhipster/jhipster-loaded) is a set of Spring Loaded plugins, that should help you when using a JHipster application: it does not just reload classes (like Spring Loaded), it also reloads and 're-wires' Spring Beans.
However, we spent a lot of time on this project, and we just stopped: it just doesn't work as well as we would like to.
Besides, there is no Java 8 support, so for us this is a no-go.
So currently your only option is to use JRebel if you want hot reload.
Keep in mind that a JHipster app starts in 4-8 seconds, so this is not such a big issue.
